src/errors.ts

```diff
@@ -59,6 +45,20 @@ export function ensureErrorType(err: unknown): Error & Partial<AzFuncError> {
     }
 }
 
+export function setErrorMessage(err: Error, message: string): Error & Partial<AzFuncError> {
```

Comment:
nit: I would rename to `trySetErrorMessage`

Reply:
Updated name